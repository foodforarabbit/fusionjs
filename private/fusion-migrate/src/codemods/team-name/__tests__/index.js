const path = require('path');
const pluginTester = require('../../../utils/codemod-test.js');

const fixtureDir = path.join(__dirname, '../__fixtures__');
const plugin = require('../plugin');

pluginTester(fixtureDir, plugin('test'));
