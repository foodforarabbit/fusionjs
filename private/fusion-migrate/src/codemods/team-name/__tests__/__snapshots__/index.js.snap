// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`team-name positive-match: positive-match 1`] = `
"import App from 'fusion-react';

const team = '';

export default (async function start(options: any = {}) {
  const root = options.root || DefaultRoot;
  const app = new App(root, options.render);
  return app;
});


      ↓ ↓ ↓ ↓ ↓ ↓

import App from 'fusion-react';
const team = 'test';
export default (async function start(options: any = {}) {
  const root = options.root || DefaultRoot;
  const app = new App(root, options.render);
  return app;
});
"
`;
